packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/SubTaskPopup/SubTaskPopup.js

```diff
@@ -51,6 +51,7 @@ function SubTaskPopup(props) {
       onOpenConfirm()
     } else {
       setSubTaskVisibility(false)
+      activeMark.setPreviousAnnotations()
```

Comment:
`setSubTaskVisibility(false)` was also clearing out previous annotations, so I've left that behaviour unchanged. I can remove this line if that wasn't intended.